kumascript/macros/HTTPSidebar.ejs

```diff
@@ -15,16 +15,14 @@ function state(section) {
 var text = mdn.localStringMap({
   'en-US': {
     'HTTP': 'HTTP',
-    'HTTPGuide': 'HTTP guide',
-    'Basics': 'Basics of HTTP',
     'Overview': 'Overview of HTTP',
     'Evolution': 'Evolution of HTTP',
-    'ResourcesURI': 'Resources and URIs',
+    'URIsResources': 'URIs and resources',
     'Identifying': 'Identifying resources on the Web',
```

Comment:
I think there's a lot of keys here no longer needed due to using the page's `short-title || title`. Essentially we only need the section titles, right?

Reply:
There are some exceptions in the `<details>` elements:

https://github.com/mdn/yari/pull/11753/files#diff-2b04bb91aa3c362eca410b8ef5a10471dcb5fa9e6672e3ee6c9d7c934d300468L359

I can remove all the keys except those? I also don't want to leave it half-refactored, but I'm happy to follow your lead here

Reply:
Yup, you can remove all that aren't referenced.

Reply:
done in https://github.com/mdn/yari/pull/11753/commits/1d9ff293ce5748e194049b319b59e7ab56b4f624